seed/views/v3/taxlots.py

```diff
@@ -518,6 +519,9 @@ def batch_delete(self, request):
         if resp[0] == 0:
             return JsonResponse({"status": "warning", "message": "No action was taken"})
 
+        # Delete orphaned mappings
+        InventoryGroupMapping.objects.filter(id__in=group_mapping_ids, taxlot__views__isnull=True).delete()
+
```

Comment:
This is triggered from the inventory list actions "delete". The function only deletes the State & View (via cascade), not the base Inventory (Property or TaxLot). Because InventoryGroups are set to cascade delete on Property or TaxLot delete, the mapping would still exist even through the orphaned Property or TaxLot